### PR TITLE
Update Novelist document format

### PR DESCRIPTION
**Pull request by leonardr**
This branch addresses https://jira.nypl.org/browse/SIMPLY-1045. There was a performance problem which turned out not to be an issue, but the NoveList API has updated their document format and we were still using the old format.

While updating the document format I discovered that we don't handle the Narrator role correctly. A book with a narrator was showing up with no author. Since Novelist considers 'narrator' an optional field, I've taken 'narrator' out altogether rather than put in the extra work to get it working -- we can always add it later and I just want to get this release out.

### CODE REVIEW

File: api/novelist.py [hunk 1]
```diff
@@ -572,24 +575,33 @@ def put_items_novelist(self, library):
...
                 content = json.loads(response.content)
+                logging.info(
+                    "Success from NoveList: %r", response.content
+                )
+            else:
+                logging.error("Data sent was: %r", data)
+                logging.error(
+                    "Error %s from NoveList: %r", response.status_code,
+                    response.content
+                )
```
Comment (leonardr) [inline api/novelist.py]: Adding this logging will help libraries communicate with NoveList.

**Comment by leonardr** [comment]
API documentation is here: http://www.novelistcollectiondata.com/api/swagger/ui/index

File: api/novelist.py [hunk 2]
```diff
@@ -477,7 +477,7 @@ def get_items_from_query(self, library):
         i1 = aliased(Identifier)
         i2 = aliased(Identifier)
         roles = list(Contributor.AUTHOR_ROLES)
-        roles.append(Contributor.NARRATOR_ROLE)
+        # roles.append(Contributor.NARRATOR_ROLE)
```
Comment (aslagle) [inline api/novelist.py]: Would probably be good to have a comment explaining why this was removed.

File: api/novelist.py [hunk 3]
```diff
@@ -539,31 +539,34 @@ def create_item_object(self, object, currentIdentifier, existingItem):
...
-                Title=title,
-                MediaType=mediaType,
-                Author=author,
-                Role=role,
-                Narrator=narrator
+                isbn=isbn,
+                title=title,
+                mediaType=mediaType,
+                author=author,
+                role=role,
```
Comment (aslagle) [inline api/novelist.py]: I don't see role in the API documentation.

Comment (leonardr) [reply api/novelist.py]: It's removed from the final document, but we keep track of it so we know whether or not we've found the primary author.

I don't think we need to keep track of it _in the document_, but I didn't want to change that.

**Pull request merged**